execution/evm/abi/core.go

```diff
@@ -169,25 +170,27 @@ func LoadPath(abiFileOrDir string) (*AbiSpec, error) {
 
 	specs := make([]*AbiSpec, 0)
 
-	err := filepath.Walk(abiFileOrDir, func(path string, fi os.FileInfo, err error) error {
-		if err != nil {
-			return fmt.Errorf("error returned while walking abiDir '%s': %v", abiFileOrDir, err)
-		}
-		ext := filepath.Ext(path)
-		if fi.IsDir() || !(ext == ".bin" || ext == ".abi") {
-			return nil
-		}
-		if err == nil {
-			abiSpc, err := ReadAbiSpecFile(path)
+	for _, dir := range strings.Split(abiFileOrDir, ":") {
```

Comment:
This should be ; on windows else "burrow vent start --abi c:\abis" is not going to work.